File: 백준/Bronze/1598. 꼬리를 무는 숫자 나열/꼬리를 무는 숫자 나열.py
a, b = map(int, input().split())

col_a = (a - 1) % 4
row_a = (a - 1) // 4

col_b = (b - 1) % 4
row_b = (b - 1) // 4

print(abs(col_a - col_b) + abs(row_a - row_b))

File: 백준/Bronze/1598. 꼬리를 무는 숫자 나열/꼬리를 무는 숫자 나열.java
import java.util.*;

class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        
        int a = sc.nextInt();
        int b = sc.nextInt();
        
        int colA = (a - 1) % 4;
        int rowA = (a - 1) / 4;
        int colB = (b - 1) % 4;
        int rowB = (b - 1) / 4;
        
        int result = Math.abs(colA - colB) + Math.abs(rowA - rowB);
        System.out.print(result);
    }
}